src/api/lib/middlewares/tokenChecker.js

```diff
@@ -17,6 +17,11 @@ export default async (ctx, next) => {
 
     try {
         ctx.user = await jwt.verify(token, config.apps.api.security.jwt.privateKey);
```

Comment:
I disagree, we do expect `ctx.user` everywhere in the API. What about setting ctx.user to only the user data ?